flocker/node/_loop.py

```diff
@@ -261,7 +274,10 @@ def output_CONVERGE(self, context):
         d = self.deployer.discover_local_state()
 
         def got_local_state(local_state):
-            self.client.callRemote(NodeStateCommand, node_state=local_state)
+            with LOG_SEND_TO_CONTROL_SERVICE(
```

Comment:
1. LOG_SEND_TO_CONTROL_SERVICE is a MessageType, should be ActionType.
2. This passes in field that is not defined there.